impl/client/src/main/doc-js/pentaho/context/IContext.jsdoc

```diff
@@ -15,14 +15,14 @@
  */
 
 /**
- * The `IContext` interface exposes contextual information of the Pentaho Web Client Platform,
- * like, for example, its
+ * The `IContext` interface allows access to the Pentaho Web Client Platform.
```

Comment:
"Allows access" would be ok, if "to contextual information" hadn't been removed — that changes the meaning.